iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -423,6 +423,7 @@ Viewer.prototype._bindHandlers = function () {
     
     $('#iframe-container .zoom-in').click(function () { viewer.zoomIn() });
     $('#iframe-container .zoom-out').click(function () { viewer.zoomOut() });
+    $('#iframe-container .print').click(function () { viewer._iframes[0].contentWindow.print() });
```

Comment:
Does this always print the first document rather than the active document of a multi-doc filing?  I suspect this should be `viewer.currentDocument().contentWindow.print()` or similar.  It looks like @paulwarren-wk is looking to remove that method in #330, but perhaps he can keep the method and just use `_currentDocumentIndex` (or you can add it back after).